docs/contributor_guide/contributing_documentation.rst

```diff
@@ -49,6 +49,14 @@ the following commands:
     ...
     start docs/_build/html/auto_examples/plot_*.html
 
+The above code block works for WindowsOS.
```

Comment:
```suggestion
The above code block works for Windows users.
```

Reply:
I have addressed the following feedback, thank you!